lib/charms/tls_certificates_interface/v1/tls_certificates.py

```diff
@@ -195,7 +202,7 @@ def _on_certificate_available(self, event: CertificateAvailableEvent) -> None:
         replicas_relation.data[self.app].update({"chain": event.chain})
         self.unit.status = ActiveStatus()
 
-    def _on_certificate_expiring(self, event: CertificateExpiringEvent) -> None:
+    def _on_certificate_expiring(self, event: Union[CertificateExpiringEvent, CertificateInvalidatedEvent]) -> None:  # noqa
```

Comment:
Please do not use bare `noqa` and add a specific code that we want to ignore.

Reply:
Fixed.